python/nav/web/webfront/urls.py

```diff
@@ -38,6 +38,11 @@
         views.toggle_dashboard_shared,
         name='dashboard-toggle-shared',
     ),
+    re_path(
+        r'^index/dashboard/toggle-subscribe/(?P<did>\d+)',
```

Comment:
```suggestion
    path(
        'index/dashboard/toggle-subscribe/<int:did>',
```

Reply:
If you stick with re_path, don't forget the final '$'.

Reply:
I used `path` :)